examples/data_preprocessing/MIMIC_III_dataset/[WIP]_prepare_MIMIC_III_data.ipynb

```diff
@@ -0,0 +1,2592 @@
+    "                             '51637','5171','7700','V066','99731','99732',\n",
+    "                             'V0382','V1261'],\n",
+    "                'Atelectasis':['7704','7705'],\n",
+    "                'Pneumothorax':['01170','01171','01172','01173',\n",
+    "                                '01174','01175','01176','5120',\n",
+    "                                '5121','51281','51282','51283',\n",
+    "                                '51289','8600','8601'],\n",
+    "                'Pleural Effusion':['51181','5119','5111'],\n",
+    "                'Pleural Other':[\"5110\"],\n",
+    "                'Fracture':['80700','80710','8190','8191','V5427',\n",
+    "                            '80701','80702','80703','80704','80705',\n",
+    "                            '80706','80707','80708','80709','81103',\n",
+    "                            '81109','81110','80711','80712','80713',\n",
+    "                            '80714','80715','80716','80717','80718',\n",
+    "                            '80719','8072','8073','8074','81000',\n",
+    "                            '81001','81002','81003','81010','81011',\n",
+    "                            '81012','81013','81100','81101','81102',\n",
+    "                            '81103','73311','81111','81112','81113',\n",
+    "                            '81119','80500','80501','80502','80503',\n",
+    "                            '80504','80505','80506','80507','80508',\n",
+    "                            '80510','80511','80512','80513','80514',\n",
+    "                            '80515','80516','80517','80518','8052',\n",
+    "                            '8053','8054','8055','8058','8059','80600',\n",
+    "                            '80601','80602','80603','80604','80605',\n",
+    "                            '80606','80607','80608','80609','80610',\n",
+    "                            '80611','80612','80613','80614','80615',\n",
+    "                            '80616','80617','80618','80619','80620',\n",
+    "                            '80621','80622','80623','80624','806025',\n",
+    "                            '80626','80627','80628','80629','80630',\n",
+    "                            '80631','80632','80633','80634','80635',\n",
+    "                            '80636','80637','80638','80639','8064',\n",
+    "                            '8065','8068','8069'],\n",
+    "                'Support Devices':['V4321','V4500','V4509','99600','99609','9961','9962','99661','V5339','99672','99674']}\n",
+    "\n",
+    "\n",
+    "# List with CheXpert Classes\n",
+    "\n",
+    "CheXpert_classes = list(CheXpert_ICD.keys())\n",
+    "CheXpert_classes = [x.lower().replace(\" \", \"_\") for x in CheXpert_classes]\n",
+    "\n",
+    "# Filter Diagnoses for CheXpert Classes only\n",
+    "\n",
+    "Diagnoses_CheXpert = Diagnoses[Diagnoses[\"ICD9_CODE\"].isin(list(chain.from_iterable(list(CheXpert_ICD.values()))))]\n",
```

Comment:
That's probably because for the short files, I just took the first 100 entries that do not necessarily have to match across the 3 data tables. It doesn't happen with the whole dataset :)